crates/dojo-lang/src/compiler.rs

```diff
@@ -131,8 +143,37 @@ impl Compiler for DojoCompiler {
         };
 
         let mut compiled_classes: HashMap<String, (Felt, ContractClass)> = HashMap::new();
+        let list_selector =  get_allowed_libfuncs_list(&props, &unit);
```

Comment:
```suggestion
        let list_selector = ListSelector::default();
```
Let's use the default list for now. :) As we mostly want to catch the pre-determined selectors form the cairo settings.